packages/integrations/mdx/src/plugins.ts

```diff
@@ -30,7 +31,6 @@ export function createMdxProcessor(mdxOptions: MdxOptions, extraOptions: MdxProc
 		rehypePlugins: getRehypePlugins(mdxOptions),
 		recmaPlugins: mdxOptions.recmaPlugins,
 		remarkRehypeOptions: mdxOptions.remarkRehype,
-		jsx: true,
```

Comment:
This is removed so the default of `jsx` is `false`, so MDX will compile directly to JS instead of needing a babel step.

Reply:
That must be quite faster I assume!